project-cli/src/templates/components/functional.mustache

```diff
@@ -10,7 +10,7 @@ import { {{interfaceName}} } from '@Interfaces';
 //#endregion Interface Imports
 
 export const {{fileName}} = (props: {{interfaceName}}.IProps): JSX.Element => (
-	<div className="{{fileName}}">
+	<div className="{{styleName}}">
```

Comment:
We tested this one and generated a functional component. After creating, we realized that `styleName`here is null. So generated functional component has empty className.